src/sources/dune.py

```diff
@@ -7,12 +7,13 @@
 from dune_client.query import QueryBase
 from pandas import DataFrame
 from sqlalchemy import BIGINT, BOOLEAN, VARCHAR, DATE, TIMESTAMP
-from sqlalchemy.dialects.postgresql import BYTEA, DOUBLE_PRECISION
+from sqlalchemy.dialects.postgresql import BYTEA, DOUBLE_PRECISION, INTEGER
 
 from src.interfaces import Source, TypedDataFrame
 
 DUNE_TO_PG: dict[str, Type[Any]] = {
     "bigint": BIGINT,
+    "integer": INTEGER,
```

Comment:
Have we seen this type being returned from Dune?

Reply:
Yes, I ran a sample query that I wrote myself, to download some data - so that I can verify uploading it after - and it returned that type. Then I got a KeyError because we didn't have it in the mapping. Thought I'd add it for safety.